test/e2e/network/service.go

```diff
@@ -3408,7 +3408,19 @@ func execAffinityTestForSessionAffinityTimeout(f *framework.Framework, cs client
 			if hosts.Len() > 1 {
 				return
 			}
-			time.Sleep(time.Duration(svcSessionAffinityTimeout) * time.Second)
+			// In some case, lvs didn't deleted the persistent connection after timeout expired,
```

Comment:
we should probably say "ipvs" in the comments to be consistent with terminology in the rest of the repo.